src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -114,6 +118,15 @@ public static int totalPloidy(final VariantContext vc, final int defaultPloidy)
         return result;
     }
 
+    /**
+     * create a genome location, given a variant context
+     * @param vc the variant context
+     * @return the genomeLoc
+     */
+    public static Locatable getLocation(final VariantContext vc) {
+        return new SimpleInterval(vc.getContig(), vc.getStart(), vc.getEnd());
```

Comment:
Delete this method -- `VariantContext` is already a `Locatable`, so this conversion is totally unnecessary.